884.uncommon-words-from-two-sentences.js

```diff
@@ -0,0 +1,28 @@
+/* URL of this problem
+    https://leetcode.com/problems/uncommon-words-from-two-sentences/
+*/
+
+/*
+ * @param {string} s1
+ * @param {string} s2
+ * @return {string[]}
+ */
+ 
+var uncommonFromSentences = function(s1, s2) {
+    const wordMap = new Map();
+    const allWords = (s1 + " " + s2).split(" ");
```

Comment:
What is your intention for `(s1 + " " + s2).split(" ")`?

Reply:
if you want to make array you can do
```suggestion
    const allWords = [s1, s2];
```